test/gtest/mha_helper.hpp

```diff
@@ -423,6 +423,33 @@ void MultiHeadAttentionfp8(const tensor<T>& q_val,
     ScaleMult(atten_heads_fp32, o_scale, multi_head_attention_fp8);
 }
 
+template <typename T>
+void MultiHeadAttentionfp16(const tensor<T>& q_val,
```

Comment:
could we rename this to  MultiHeadFwdAttentionfp16. May be we need to do same for fp8 version too.